src/app/hm-admin/helper/urlHelper.ts

```diff
@@ -0,0 +1,58 @@
+export const urlHelper = {
+
+  /**
+   * get path with params
+   *
+   * @param {string} url
+   * @param {string} params
+   *
+   * @returns {string}
+   */
+  getPath(url: string, addiionnalParams: any): string {
+    const [path, ] = url.split('?');
+
+    return path + urlHelper.getParams(url, addiionnalParams);
+  },
+
+  /**
+   * transform parameter's object to URL parameters
+   *
+   * @param {string} url
+   * @param {string} params
+   *
+   * @returns {string}
+   */
+  getParams(url: string, params: any): string {
```

Comment:
We are talking about this right ? https://angular.io/docs/ts/latest/api/http/index/URLSearchParams-class.html